source/guide_seafile.rst

```diff
@@ -0,0 +1,271 @@
+.. highlight:: console
+
+.. author:: Finn <mail@f1nn.eu>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/seafile.png
+      :align: center
+
+##########
+Seafile
+##########
+
+Seafile is an enterprise file hosting platform with high reliability and performance. Put files on your own server. Sync and share files across different devices, or access all the files as a virtual disk.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * Python_ and its package manager pip
+  * MySQL_
+  * domains_
+
+License
+=======
+
+All relevant legal information can be found here
+
+  * https://www.seafile.com/en/product/private_server/
+
+Prerequisites
+=============
+
+You'll need your MySQL credentials_. Get them with ``my_print_defaults``:
+
+::
+
+ [isabell@stardust ~]$ my_print_defaults client
+ --default-character-set=utf8mb4
+ --user=isabell
+ --password=MySuperSecretPassword
+ [isabell@stardust ~]$
+
+Your URL needs to be setup:
+
+::
+
+ [isabell@stardust ~]$ uberspace web domain list
+ isabell.uber.space
+ [isabell@stardust ~]$
+
+Installation
+============
+
+Step 1
+------
+
+Download the seafile server and extract it to a folder called seafile.
+
+::
+
+ [isabell@stardust ~]$ wget https://download.seadrive.org/seafile-server_6.3.4_x86-64.tar.gz
+ [isabell@stardust ~]$ mkdir seafile
+ [isabell@stardust ~]$ mv seafile-server_* seafile
+ [isabell@stardust ~]$ cd seafile/
+ [isabell@stardust ~]$ tar -xzf seafile-server_*
+ [isabell@stardust ~]$ mkdir installed
+ [isabell@stardust ~]$ mv seafile-server_* installed/
+ [isabell@stardust ~]$
+
+Step 2
+------
+
+Install the required python libraries.
+
+::
+
+ [isabell@stardust ~]$ curl --silent http://effbot.org/media/downloads/Imaging-1.1.7.tar.gz | tar -xzf -
+ [isabell@stardust ~]$ cd Imaging-1.1.7
+ [isabell@stardust ~]$ perl -pi -e 's|/usr/lib|/usr/lib64|g' setup.py
+ [isabell@stardust ~]$ python2.7 setup.py install --user
+ [isabell@stardust ~]$ cd ..
+ [isabell@stardust ~]$ rm -rf Imaging-1.1.7
+ [isabell@stardust ~]$
+
+Step 3
+------
+
+Create the needed MySQL databases as the installer file won't work on uberspace.
+
+.. warning:: Replace ``isabell`` with your username!
+
+::
+
+ [isabell@stardust ~]$ mysql
+ MariaDB [(none)]>     create database `isabell_ccnet-db` character set = 'utf8';
```

Comment:
our MySQL defaults to `utf8mb4`, which enables the full utf8-set, including emojis and other fun things. Does seafile really only support `utf8`?

Reply:
> 
> 
> our MySQL defaults to `utf8mb4`, which enables the full utf8-set, including emojis and other fun things. Does seafile really only support `utf8`?

I'm currently not sure about that. I haven't tried that yet, but I guess it would work. I'm using the create queries from the [official installation guide](https://manual.seafile.com/deploy/using_mysql.html).

Reply:
we'll stick to the official guide then. thanks for taking a look!